pkg/test/case.go

```diff
@@ -370,6 +370,16 @@ func (t *Case) Run(test *testing.T, ts *report.Testsuite) {
 		tc.Assertions += len(testStep.Asserts)
 		tc.Assertions += len(testStep.Errors)
 
+		if testStep.LazyLoadKubeConfig {
+			cl, err = newClient(testStep.Kubeconfig)(false)
+			if err != nil {
+				setupReport.Failure = report.NewFailure(err.Error(), nil)
+				ts.AddTestcase(setupReport)
+				test.Fatal(err)
```

Comment:
At this point the `setupReport` is already added (line 357).
I think on error here we should instead do something similar to what happens in lines 385-390.